### ISSUE TITLE
Sample Projects missing Files?

### ISSUE THREAD

**Issue by amrit7** [CommunityToolkit/MVVM-Samples#117]
Hi,

I downloaded the ZIP file for MVVM-Samples from GitHub and having the following issues, related to empty folders / missing files:

1. This folder is empty in the ZIP file: MVVM-Samples-master\msdocs-communitytoolkit
2. The Project MvvmSampleUWP does not build as it cannot find any files under: \msdocs-communitytoolkit\docs\mvvm

For example - I can see the Project is trying to find, but is unable to do so:

    <Content Include="..\..\msdocs-communitytoolkit\docs\mvvm\generators\Overview.md">
      <Link>Assets\docs\generators\Overview.md</Link>
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>

Please can you review and check the samples are working.

Thanks

**Comment by michael-hawker** [comment]
@amrit7 there's a submodule, so you need to be sure to clone the submodules as well when working in the repo. @Sergio0694 probably should add a line in the readme about how to do the checkout?

**Comment by michael-hawker** [comment]
@amrit7 yup, the submodule is denoted with the arrowed folder, as seen here:

![image](https://user-images.githubusercontent.com/24302614/233125589-da5e1dac-866e-4405-8bb1-c67790905760.png)

Submodules allows one git repository to include the source of another. This is useful for a variety of reasons, but here it's because rather than copying files from the documentation of the other repository to use within the app, we can include a submodule to directly reference them instead. It make it easier to keep the app in-sync with the documentation.
